public/index.html

```diff
@@ -11,7 +11,7 @@
       property="og:description"
       content="Built by the Micro:bit Educational Foundation and the global Python Community."
     />
-    <meta property="og:image" content="%PUBLIC_URL%/social.png"/>
+    <meta property="og:image" content="https://python.microbit.org/%PUBLIC_URL%social.png"/>
```

Comment:
Missing slash between %PUBLIC_URL% and social?

Reply:
It really does seem unreasonable that og:image has to be absolute.